src/app/providers/provider-form/azuredevops/azuredevops.component.ts

```diff
@@ -29,9 +33,15 @@ export class AzureDevOpsComponent {
     };
 
     this.azureService.azuredevopsPost(providerData)
-      .pipe(
-        finalize(() => this.providersService.triggerCloseForm(this.providerForm)),
-      )
-      .subscribe();
+      .pipe(first())
+      .subscribe(() => {
+          this.providersService.triggerCloseForm(this.providerForm);
+          this.providerForm.enable();
```

Comment:
Should this reenabling maybe be placed in a "finally" clause? Like in the third argument of `.subscribe()`?

Reply:
Good point. Implemented.